src/index.html

```diff
@@ -22,6 +22,367 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="https://www.google.com"
+      >
+        <img
+          class="header__logo-image"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="header-nav">
+        <ul class="header-nav__list">
+          <li class="header-nav__item">
+            <a
+              class="header-nav__item-link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header-nav__item">
+            <a
+              class="header-nav__item-link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header-nav__item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="header-nav__item">
```